contracts/pair_stable_bluna/src/state.rs

```diff
@@ -0,0 +1,32 @@
+use astroport::asset::PairInfo;
+use cosmwasm_std::{Addr, Decimal256, Uint128};
+use cw_storage_plus::{Item, Map};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+/// ## Description
+/// This structure describes the main control config of pair stable.
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct Config {
+    /// the type of pair info available in [`PairInfo`]
+    pub pair_info: PairInfo,
+    /// the factory contract address
+    pub factory_addr: Addr,
+    /// The last time block
+    pub block_time_last: u64,
+    /// The last cumulative price 0 asset in pool
+    pub price0_cumulative_last: Uint128,
+    /// The last cumulative price 1 asset in pool
+    pub price1_cumulative_last: Uint128,
+    pub init_amp: u64,
+    pub init_amp_time: u64,
+    pub next_amp: u64,
+    pub next_amp_time: u64,
+    pub bluna_rewarder: Addr,
+}
+
+pub const CONFIG: Item<Config> = Item::new("config");
+
+pub const BLUNA_REWARD_HOLDER: Item<Addr> = Item::new("bluna_reward_holder");
```

Comment:
Do we need a separate storage item for this and BLUNA_REWARD_GLOBAL_INDEX? Maybe let's put to config as well?

Reply:
Ok, but BLUNA_REWARD_GLOBAL_INDEX is more like state